config/assets/default.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
         'public/lib/angular-ui-router/release/angular-ui-router.js',
         'public/lib/angular-ui-utils/ui-utils.js',
         'public/lib/angular-bootstrap/ui-bootstrap-tpls.js',
```

Comment:
If you don't have this correction then you must remove dependency injection `['ui.bootstrap']` and the dropdowns won't work... but you know no big deal otherwise you will get error [ui.bootstrap] is not defined.